src/Shell/WorkerShell.php

```diff
@@ -44,11 +47,28 @@ public function main()
             });
         }
 
-        while($worker->work()) {
+        while ($worker->work()) {
             if ($worker->returnCode() != GEARMAN_SUCCESS) {
                 echo "return_code: " . $worker->returnCode() . "\n";
                 break;
             }
         }
     }
+
+    /**
+     * Returns an array with all configured jobs.
+     *
+     * @throws \Cake\Core\Exception\Exception
+     * @return array Hash with jobs as found in Configure
+     */
+    protected function getJobs()
+    {
+        $jobs = Configure::read('Gearman.Jobs');
```

Comment:
Shouldn't it also check for `Jobs` because that's what used before? It would be a BC breaking change otherwise.